features/telemetry/protocols/netflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/netflow5/Netflow5ConverterTest.java

```diff
@@ -95,8 +93,8 @@ public void canParseNetflow5Flows() {
         MatcherAssert.assertThat(flow.getPackets(), equalTo(5L));
         MatcherAssert.assertThat(flow.getDirection(), equalTo(Flow.Direction.INGRESS));
         MatcherAssert.assertThat(flow.getNextHop(), equalTo("0.0.0.0"));
-        MatcherAssert.assertThat(flow.getNextHopHostname(), equalTo(Optional.empty()));
-        MatcherAssert.assertThat(flow.getVlan(), nullValue());
+        MatcherAssert.assertThat(flow.getNextHopHostname(), equalTo(Optional.of("")));
+        MatcherAssert.assertThat(flow.getVlan(), equalTo(0));
```

Comment:
VLAN 0 is not equal to no VLAN

Reply:
yeah, proto has primitives and never returns null  even for Strings.  need to find a way to differentiate between actual values and not having a value.